kci_build

```diff
@@ -401,6 +403,16 @@ class cmd_push_kernel(Command):
         install = kernelci.build.Step.get_install_path(args.kdir, args.output)
         meta = kernelci.build.Metadata(install)
         publish_path = meta.get('bmeta', 'kernel', 'publish_path')
+        defconfig = meta.get('bmeta', 'kernel', 'defconfig')
+        # gki_defconfig is exceptional and generate hundreds of megabytes
+        # several files, which require compression
+        # to workaround current upload limits
+        if (defconfig == "gki_defconfig"):
+            kernel_fname = meta.get('bmeta', 'kernel', 'image')
+            compressed_artifacts = ["logs/kernel.log"]
+            compressed_artifacts.append("kernel/"+kernel_fname)
+            for e in compressed_artifacts:
+                subprocess.run(["xz", "-9", os.path.join(install, e)])
```

Comment:
It would be a bit nicer to use the `lzma` package: https://docs.python.org/3/library/lzma.html

Reply:
Added. At moment it will read() whole file in memory at once, but as it is expected to be pod with 30G of ram and there is no other processes at moment i think it should be fine, as files are expected max 1-2Gb